README.md

```diff
@@ -47,6 +47,15 @@ scalene_profiler.start()
 scalene_profiler.stop()
 ```
 
+To use Scalene as a decorator, import `scalene_redirect_profile`
+```Python
+from scalene.scalene_profiler import scalene_redirect_profile
```

Comment:
Can just replace all this with `@profile` (no need to import `scalene_redirect_profile`, etc.).

Reply:
Bump @Birdi7 - do you want to make a new PR or close this one?

Reply:
I'll edit this PR, and bump you. Plan to do this during the next 5 days. Thank you for the bump :)

Reply:
@emeryberger I've edited the PR to describe `@profile` usage